### PR TITLE
Typos from Perl5 (2008-2010)

### PR DESCRIPTION
**Pull request by Fourchaux**
Some typos from https://perl5.git.perl.org/perl.git/search/HEAD?s=typo;st=free

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -5127,6 +5143,7 @@ effctive->effective
 effeciency->efficiency
 effecient->efficient
 effeciently->efficiently
+effecting->affecting
```
Comment (larsoner) [inline codespell_lib/data/dictionary.txt]: `effecting` is valid (and not entirely uncommon), let's remove this

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -9763,6 +9802,7 @@ optiional->optional
 optimasation->optimization
 optimazation->optimization
 optimiality->optimality
+optimier->optimiser
```
Comment (larsoner) [inline codespell_lib/data/dictionary.txt]: probably should suggest `optimizer` first

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -10121,6 +10165,7 @@ paremeter->parameter
 paremeters->parameters
 paremter->parameter
 paremters->parameters
+paren->parent
```
Comment (larsoner) [inline codespell_lib/data/dictionary.txt]: `paren` is fine (used often as "a parenthesis"), let's remove this

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -12614,6 +12667,7 @@ sergent->sergeant
 serialiazation->serialization
 serie->series
 serieses->series
+seriopts->scripts
```
Comment (larsoner) [inline codespell_lib/data/dictionary.txt]: this one seems a bit far?

Comment (Fourchaux) [reply codespell_lib/data/dictionary.txt]: Yes, my bad!
I mean: scriopts->scripts

**Pull request merged**